install/resources/kafka/cluster-example.kafka.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: kafka.strimzi.io/v1beta2
+kind: Kafka
+metadata:
+  name: cluster-example
+spec:
+  entityOperator:
+    topicOperator: {}
+    userOperator: {}
+  kafka:
+    authorization:
+      type: simple
+    config:
+      allow.everyone.if.no.acl.found: 'true'
+      default.replication.factor: 3
+      inter.broker.protocol.version: '3.6'
+      min.insync.replicas: 2
+      offsets.topic.replication.factor: 3
+      transaction.state.log.min.isr: 2
+      transaction.state.log.replication.factor: 3
+    listeners:
+      - name: secure
+        port: 9093
+        tls: true
+        type: ${LISTENER_TYPE}
```

Comment:
We can use any listener type supported by AMQ Streams?  We don't support ingress downstream, only on Strimzi. Route is expected downstream.